1-js/02-first-steps/12-nullish-coalescing-operator/article.md

```diff
@@ -17,7 +17,7 @@ The nullish coalescing operator isn't anything completely new. It's just a nice
 We can rewrite `result = a ?? b` using the operators that we already know, like this:
 
 ```js
-result = (a !== null && a !== undefined) ? a : b;
+result = (a != null && a != undefined) ? a : b;
```

Comment:
both the old and new diff mean the same - the !== is not a typo, it is a separate operator in JavaScript https://javascript.info/comparison#strict-equality